api/src/main/java/uk/gov/dvsa/mot/trade/api/RfrAndAdvisoryItem.java

```diff
@@ -30,4 +31,15 @@ public RfrAndAdvisoryItem setType(String type) {
         this.type = type;
         return this;
     }
+
+    public Boolean getDangerous() {
+
+        return dangerous;
+    }
+
+    public RfrAndAdvisoryItem setDangerous(Boolean dangerous) {
+
+        this.dangerous = dangerous;
+        return this;
```

Comment:
This class provides a fluent API which doesn't seem consistent with other classes in this application